xcodeproj/internal/output_files.bzl

```
@@ -1,6 +1,7 @@
 """Module containing functions dealing with target output files."""
 
 load(":filelists.bzl", "filelists")
+load(":frozen_constants.bzl", "EMPTY_DEPSET")
```

Surprised this works:
```suggestion
load(":frozen_constants.bzl", "EMPTY_DEPSET", "NONE_LIST")
```